dali/operators/reader/coco_reader_op_test.cc

```diff
@@ -493,62 +517,78 @@ TEST_F(CocoReaderTest, LtrbRatioSkipEmpty) {
   this->RunTest(true, true, true);
 }
 
-TEST_F(CocoReaderTest, Masks) {
+TEST_F(CocoReaderTest, PolygonMasks) {
   this->RunTest(false, false, false, true);
 }
 
+TEST_F(CocoReaderTest, PolygonMasksLegacy) {
+  this->RunTest(false, false, false, false, true);
+}
+
 TEST_F(CocoReaderTest, PixelwiseMasks) {
   this->file_root_ = dali::testing::dali_extra_path() + "/db/coco_pixelwise/images";
   this->annotations_filename_ = dali::testing::dali_extra_path() +
                                       "/db/coco_pixelwise/instances.json";
   int expected_size = 6;
-  Pipeline pipe(expected_size, 1, 0);
-  pipe.AddOperator(
+  int kSeed = 12345;
+  Pipeline pipe1(expected_size, 1, 0, kSeed);
+  pipe1.AddOperator(
     CocoReaderOpSpec(false, true)
-    .AddArg("masks", false)
-    .AddArg("pixelwise_masks", true)
-    .AddArg("dump_meta_files", true)
-    .AddArg("dump_meta_files_path", "/tmp/")
-    ,
+    .AddArg("save_preprocessed_annotations", true)
+    .AddArg("save_preprocessed_annotations_dir", "/tmp/"),
```

Comment:
I wonder if we should use [mkdtmp](https://man7.org/linux/man-pages/man3/mkdtemp.3.html) instead of hardcode to just /tmp/.

Reply:
done